src/app/admin/pages/users/users.component.scss

```diff
@@ -1,11 +1,20 @@
 @import '~_vars.scss';
 
+.users {
+  max-width: 100%;
+  position: relative;
+  margin: auto;
+  box-sizing: border-box;
+  min-height: 500px;
+  overflow: hidden;
+  margin-top: 40px;
+  margin-bottom: 10px;
+}
+
 .user-cards {
   margin-top: 30px;
-  display: flex;
-  flex-direction: row;
-  flex-wrap: wrap;
-  justify-content: flex-start;
+  display: grid;
+  grid-template-columns: repeat(5, 20%);
```

Comment:
Instead of using a percentage here, can we use a minmax function? On smaller screens, 20% of the body might be far too small. Likewise, on bigger screens 20% might be far too big.